### DISCUSSION TITLE
Remote executor and cache

### DISCUSSION THREAD

**Discussion by tsmarques** [bazelbuild/bazel#22030] [Q&A] [answered]
Hi,

sorry if this is documented or has been answered somewhere but I haven't found a clear answer. I'm experimenting with remote executor and remote cache (--remote_executor and --remote_cache) and was wondering: Is there a point or benefit in running with both flags active (using the same endpoint)? e.g.

> bazel build --remote_executor=<...>  --remote_cache=<...>

What I'm wondering specifically is if running only with _--remote_executor_ also builds a remote cache that can be used by others the same way as --remote_cache

thank you!

**Comment by tjgq** [comment, marked as answer]
The --remote_cache flag is redundant when set to the same value as the
--remote_execution flag. The latter is sufficient to populate the cache for
use by other builds.

**Reply by tsmarques** [reply]
Thank you for the clarification!
